backend/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==2.2.3
```

Comment:
Can we move this file out into a general requirements.txt file -- its going to make synchornization between ML and backend easier.

Reply:
backend ml stuff should go under `backend/` folder